pom.xml

```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.google.sps</groupId>
-  <artifactId>jack-sps-summer-22-team-36</artifactId>
+  <artifactId>summer22-sps-36</artifactId>
```

Comment:
Should we add pom to the.gitignore so we don't have to keep switching it?

Reply:
yeah, i think so